src/__tests__/ceramic_cas.ts

```diff
@@ -27,14 +31,10 @@ describe('Ceramic<->CAS integration', () => {
     let ceramic: CeramicApi
 
     beforeAll(async () => {
-        const services: Services = await buildServicesFromConfig();
+        const services: Services = global.services;
```

Comment:
Could be simply `ceramic = global.services.ceramic` for easier readability maybe?